addon/tests/helpers/with-feature.js

```diff
@@ -1,8 +1,8 @@
 import Ember from 'ember';
+import features from 'ember-feature-flags/features';
 
 export function withFeature( featureName ){
-  window.Features = window.Features || {};
-  window.Features[featureName] = true;
+  features.set(featureName, true);
```

Comment:
From @aexmachina: Re. "turn a feature on for a test but if it was off at the first render", yes this is not good. We get around it by providing the features in the call to `startApp()`, and with the unit tests calling `setup()` with the full set of features.

There's issues with both approaches:

1. Keeping the flags in the module: this is convenient, but you get leakage between tests, if you don't call `setup()` then your test will run with the features from the last test.
1. Making it a service: this avoids leakage, but then it's difficult for unit tests: you need to have `needs: ['service:features']` and manage the injections manually. We could provide a helper for the injections:

    ```js
    import {setupFeatures} from 'ember-feature-flags';
    describeComponent('something', 'Some component', {
      needs: ['service:features']
    }, function() {
      setupFeatures({featureA: true, featureB: false}, this.container);
      this.render();
      // ...
    })
    ```